tests/integrations/docker/test_Docker__run.py

```diff
@@ -81,32 +71,27 @@ def test_simple_call_with_path_arg(mock_docker, tmp_path, capsys):
 )
 def test_simple_verbose_call(mock_docker, tmp_path, capsys):
     "If verbosity is turned out, there is output"
-    mock_docker.command.verbosity = 2
+    mock_docker.command.logger = Log(verbosity=2)
 
-    mock_docker.run(['hello', 'world'])
+    mock_docker.run(["hello", "world"])
 
     mock_docker._subprocess._subprocess.run.assert_called_with(
         [
-            'docker',
-            'run', '--tty',
-            '--volume', '{platform_path}:/app:z'.format(
-                platform_path=tmp_path / 'platform'
-            ),
-            '--volume', '{dot_briefcase_path}:/home/brutus/.briefcase:z'.format(
-                dot_briefcase_path=tmp_path / '.briefcase'
-            ),
-            'briefcase/com.example.myapp:py3.X',
-            'hello',
-            'world',
+            "docker",
+            "run", "--tty",
+            "--volume", f"{tmp_path / 'platform'}:/app:z",
+            "--volume", f"{tmp_path / '.briefcase'}:/home/brutus/.briefcase:z",
+            "briefcase/com.example.myapp:py3.X",
+            "hello",
+            "world",
         ]
     )
     assert capsys.readouterr().out == (
-        ">>> docker run --tty "
-        "--volume {platform_path}:/app:z "
-        "--volume {dot_briefcase_path}:/home/brutus/.briefcase:z "
+        ">>> \n"
+        ">>> Running Command:\n"
```

Comment:
The extra newlines on these two intrigue me - where do they come from? `subprocess.run` will do a clearing newline, but that doesn't seem to match what is being output here

Reply:
This also goes back to the additional new line I introduced via `Subprocess._log_command()` and talked about above.